R/data_type.R

```diff
@@ -27,14 +27,14 @@
 #'   DrugName = c("DRUG_10", "DRUG_8"),
 #'   CellLineName = "CELL1"
 #' )
-#' input_df <- as.data.frame(rbind(ctrl_df, trt_df))
+#' input_df <- data.table::setDT(as.data.frame(rbind(ctrl_df, trt_df)))
```

Comment:
`setDT` modifies objects by reference, so here you need to use just: `input_df <- data.table::data.table(rbind(ctrl_df, trt_df))`